packages/tags/stories/examples/Default.tsx

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Story } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { Tag } from '@zendeskgarden/react-tags';
+import { Col, Grid, Row } from '@zendeskgarden/react-grid';
+
+interface IDefaultStoryProps {
+  text: string;
+  hue: string;
+  size: 'small' | 'medium' | 'large';
+  shape: 'default' | 'pill' | 'round';
+  isRegularWeight: boolean;
+  includeAvatar: boolean;
+  includeClose: boolean;
+}
+
+export const Default: Story<IDefaultStoryProps> = ({
+  text,
+  hue,
+  size,
+  shape,
+  isRegularWeight,
+  includeAvatar,
+  includeClose
+}) => (
+  <Grid>
+    <Row>
+      <Col textAlign="center">
+        <Tag
+          hue={hue === 'default' ? undefined : hue}
+          isPill={shape === 'pill'}
+          isRound={shape === 'round'}
+          isRegular={isRegularWeight}
+          size={size}
+          tabIndex={0}
+        >
+          {includeAvatar && (
+            <Tag.Avatar>
+              <img alt="" src={`images/avatar-${Math.floor(Math.random() * 70 + 1)}.png`} />
```

Comment:
Let's simplify the code (and deployment size) by selecting a single, fixed avatar.

Reply:
Updated to a single image.